src/components/navBar/demoBasicUsage/index.html

```diff
@@ -0,0 +1,11 @@
+<div ng-controller="AppCtrl" ng-cloak>
+  <md-content class="md-padding">
+    <md-nav-bar selected-link-name="currentLink" nav-bar-aria-label="navigation links">
+      <md-nav-link nav-click="goto('page1')" link-name="page1">Page One</md-nav-link>
```

Comment:
* Why not `md-nav-item`... which is similar to the APIs in other components. 
* Why no use `name` instead of `link-name`?